pkgs/development/tools/godef/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoPackage rec {
   name = "godef-${version}";
-  version = "20170920-${stdenv.lib.strings.substring 0 7 rev}";
-  rev = "b692db1de5229d4248e23c41736b431eb665615d";
+  version = "20180911-${stdenv.lib.strings.substring 0 7 rev}";
```

Comment:
Please remove the hash from the version number.

Reply:
Yeah, I figured I should update that to match contemporary practices. I'll do that when I'm back home.